### ISSUE TITLE
Error: access denied

### ISSUE THREAD

**Issue by notcreeperdude** [Stiphr/Steam-Bulk-Trade-Bot#2]
Hi, running the code i get a successful "logged into steam" followed by an immediate Error: Access denied. Attached screenshot below:
Any idea as to what causes this? I've checked that the files are there and tried running as admin, also tried VPN to ensure its not my connection.
![image](https://github.com/Stiphr/Steam-Bulk-Trade-Bot/assets/75228915/11e32011-64b5-462a-8eb0-9dd5e3dc2617)

**Comment by Stiphr** [comment]
Make sure accounts format look like this:

username:password:2fa_token:identity_secret
username:password:2fa_token:identity_secret
username:password:2fa_token:identity_secret

**Comment by notcreeperdude** [comment]
> Make sure accounts format look like this:
> 
> username:password:2fa_token:identity_secret username:password:2fa_token:identity_secret username:password:2fa_token:identity_secret

is token_gid the 2fa_token? Also i got the identity_secret from my SteamDesktopAuthentictor mafiles.

**Comment by Stiphr** [comment]
> > Make sure accounts format look like this:
> > username:password:2fa_token:identity_secret username:password:2fa_token:identity_secret username:password:2fa_token:identity_secret
> 
> is token_gid the 2fa_token? Also i got the identity_secret from my SteamDesktopAuthentictor mafiles.

No, shared_secret is your 2fa key: `"shared_secret":"a/qWNvl2Kii4Wi/PXUIG6Au7ccL="`
